packages/solid/components/Keyboard/formats.ts

```diff
@@ -0,0 +1,1238 @@
+import type { KeyboardFormat } from './Keyboard.types.js';
+
+const dialpad = {
+  default: [['1', '2', '3'], ['4', '5', '6'], ['7', '8', '9'], ['0']]
+};
+
+const dialpadExtended = {
+  default: [
+    ['1', '2', '3'],
+    ['4', '5', '6'],
+    ['7', '8', '9'],
+    ['0'],
+    [
+      {
+        title: 'Delete',
+        size: 'lg',
+        keyId: 'delete',
+        announce: 'delete, button'
+      }
+    ]
```

Comment:
Any reason why we wouldn't want the 0 and delete keys both in row 4 with delete at size md? I realize LUI dialpad extended wasn't like this, but on a previous project I had to create a custom keyboard because this UX was preferred.

![Screenshot 2024-08-19 at 3 50 19 PM](https://github.com/user-attachments/assets/956c2a2f-60d5-4c36-89cb-b5247e31da3d)

Reply:
no reason, i just pulled in the existing formats. this is a great call, I'll get that format updated